python/cudf/cudf/core/algorithms.py

```diff
@@ -58,9 +58,7 @@ def factorize(values, sort=False, na_sentinel=-1, size_hint=None):
 
     cats = values._column.dropna().unique().astype(values.dtype)
 
-    name = values.name  # label_encoding mutates self.name
     labels = values._label_encoding(cats=cats, na_sentinel=na_sentinel).values
```

Comment:
I'm confused, how was this working before? It looks like `values` is a `Column` here, not a `Series`, but the `Column` method didn't exist before this PR right? Is this dead code somehow?

Reply:
Oh sorry I missed that the line above assigned the output to `cats`, not back to `values`. Never mind.